src/crimson/os/seastore/random_block_manager/nvme_block_device.cc

```diff
@@ -22,13 +22,29 @@ namespace {
 
 namespace crimson::os::seastore::random_block_device::nvme {
 
+NVMeBlockDevice::mkfs_ret NVMeBlockDevice::mkfs(device_config_t config) {
+  using crimson::common::get_conf;
+  shard_devices.local().set_journal_size(
+    get_conf<Option::size_t>("seastore_cbjournal_size"));
+  return shard_devices.local().do_primary_mkfs(config, seastar::smp::count
```

Comment:
needn't transfer seastar::smp::count, it can get in do_primay_mkfs.

Reply:
To reuse do_primary_mkfs() in EphemeralRBMDevice, do_primary_mkfs() has the number of shards as an input argument. Note that the current Ephemeral test only supports a single shard case.